tests/testthat/test-slice-chop.R

```diff
@@ -580,6 +580,24 @@ test_that("name repair is respected and happens after ordering according to `ind
   expect_named(list_unchop(x, indices, name_repair = "unique"), c("a...1", "a...2"))
 })
 
+test_that("list_unchop() can repair names quietly", {
+  local_name_repair_verbose()
+
+  x <- c(x = "a", x = "b", x = "c")
+  indices <- list(2, c(3, 1))
+  expect_snapshot({
+    res <- list_unchop(vec_chop(x, indices), indices, name_repair = "unique_quiet")
```

Comment:
```suggestion
    res <- list_unchop(vec_chop(x, indices), indices = indices, name_repair = "unique_quiet")
```

Reply:
I'm forcing this to be a named argument in #1715 so this should make it a little simpler to merge those prs in any order